src/assisted_installer_controller/assisted_installer_controller.go

```diff
@@ -1075,6 +1078,67 @@ func (c controller) logClusterOperatorsStatus() {
 	}
 }
 
+// This is temporary function that should validate router state only in case of failure
+// It will patch router to add access logs
+// It will run http router health check to see if router is healthy on host network
+func (c controller) logRouterStatus() {
+	if (!c.Status.HasError() && !c.Status.HasOperatorError()) || c.HighAvailabilityMode != models.ClusterHighAvailabilityModeNone {
```

Comment:
Why do we need to check both?
(!c.Status.HasError() && !c.Status.HasOperatorError())